diagnostic_updater/include/diagnostic_updater/DiagnosticStatusWrapper.hpp

```diff
@@ -257,19 +258,21 @@ inline void DiagnosticStatusWrapper::add<bool>(const std::string & key, const bo
 
 // Need to place addf after DiagnosticStatusWrapper::add<std::string> or
 // gcc complains that the specialization occurs after instatiation.
-inline void DiagnosticStatusWrapper::addf(const std::string & key, const char * format, ...) // In practice format will always be a char *
+inline void
+DiagnosticStatusWrapper::addf(
+  const std::string & key, const char * format,
+  ...)                              // In practice format will always be a char *
 {
   va_list va;
-  char buff[1000];   // @todo This could be done more elegantly.
+  char buff[1000];  // @todo This could be done more elegantly.
   va_start(va, format);
-  if (vsnprintf(buff, 1000, format, va) >= 1000) {
-    ROS_DEBUG("Really long string in DiagnosticStatusWrapper::addf, it was truncated.");
+  if (vsnprintf(buff, sizeof(buff), format, va) >= 1000) {
```

Comment:
If this is going to change, it can be `if (vsnprintf(buff, sizeof(buff), format, va) >= sizeof(buff) {`